src/List/ListItemText.js

```diff
@@ -14,6 +14,9 @@ export const styleSheet = createStyleSheet('ListItemText', () => {
         paddingLeft: 0,
       },
     },
+    inset: {
+      paddingLeft: '56px !important',
```

Comment:
Added to bypass the `&:first-child` specificity.
@nathanmarks Do you see a better approach?